src/test/java/com/mongodb/jdbc/MongoSQLTranslateLibTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2024-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.jdbc;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MongoSQLTranslateLibTest {
+    @BeforeEach
+    void setup() throws Exception {
```

Comment:
Can you add a comment to explain that this because we need to reset the flag between each test?
I had to read this twice to realize that the setup was needed because the first test was loading the lib and setting the flag to true (hopefully).

Reply:
Sure I'll add a comment.  It's also because I have `initializeMongoSqlTranslateLibrary();` in the static initialization block so it also gets set there.